elements/map/src/generate.ts

```diff
@@ -1,10 +1,20 @@
 import * as olLayers from "ol/layer";
 import * as olSources from "ol/source";
 import * as olFormats from "ol/format";
+import STAC from "ol-stac/layer/STAC";
```

Comment:
```suggestion
import STAC from "ol-stac";
```

Reply:
This does not seem to work, while the properties work perfectly fine now via `import STAC from "ol-stac/layer/STAC";`

Reply:
Hmm, thanks. I'll check it.

Reply:
I think it's this line that still creates a link to `index.js` https://github.com/m-mohr/ol-stac/blob/main/tasks/prepare-package.js#L13

Reply:
`beta4` fixed it! :tada: